technic/doc/tools.md

```diff
@@ -42,6 +42,30 @@ The higher levels of the mining drill allow for drilling more blocks at once, su
 
 This electric tool is a substantial upgrade above the pickaxe that you start mining with. The laser works by drilling directly forward 7 blocks making a path that's roughly 1-block in diameter. The laser is electric, requiring charging in a battery box (any voltage). As soon as an LV electrical system is set up, this is a great next step.
 
+## Multimeter
+
+<img src="../textures/technic_multimeter.png"/>
+
+This electric tool is used for quick lookup at an electric network. It works by clicking on an electric node (cable, machine, generator, switchting station), and gives values about : the position of the attached switching station, its current status, the lag, the batteries, the supply and the demand, and the total of each type of node.
+
+| Property         | Example Value | Unit       | Meaning                                                       |
+|------------------|---------------|------------|---------------------------------------------------------------|
+| Ref. point       | 61, 19, 62    | Coordinate | The location of the switching station attached to the network |
+| Activated        | yes           | active     | The current state of the network                              |
+| Timeout          | 1800.0        | s          | The timeout of the network                                    |
+| Lag              | 0.60          | ms         | The lag generated by the network                              |
+| Skip             | 0             | cycles     | How many cycles are skipped                                   |
+| Supply           | 6000          | EU         | How many power is supplied by generators                      |
+| Demand           | 4000          | EU         | How many power is required by consumers                       |
+| Battery charge   | 2911956       | EU         | The total of energy stored in batteries                       |
+| Battery charge   | 97.07         | %          | The energy stored per the maximum that can be stored          |
+| Battery capacity | 3000000       | EU         | The maximum amount of energy that can be stored in batteries  |
+| Nodes            | 47            | count      | The number of nodes of the network                            |
```

Comment:
```suggestion
| Nodes            | 47            | count      | The number of nodes of the network, excluding switch stations |
```

Reply:
For multimeter, would probably be good to have switching station count as a separate measurement and also make waypoint functionality display waypoints for switching stations.
Not for this pull request but good update now there's actually easy straightforward way to get/store positions of all connected switching stations.